src/utils/filters.py

```diff
@@ -0,0 +1,21 @@
+import os
+from functools import reduce
+
+
+def any_filter_combinator(*filters):
+    return reduce(lambda acc, f: lambda x: acc(x) or f(x), filters, lambda _: False)
+
+
+def all_filter_combinator(*filters):
+    return reduce(lambda acc, f: lambda x: acc(x) and f(x), filters, lambda _: True)
+
+
+def file_has_extension(file, extensions=None):
+    return extensions is None or any([file.endswith(ext) for ext in extensions])
+
+
+def file_has_size(file, left_bound=None, right_bound=None):
```

Comment:
Why file size, not stored graph size?

Reply:
I think there should be both options. There are problems with the size of graphs. Exactly to compute it for rdf files at first we need to parse this files. This takes a long time. So maybe the graph size should be approximated by the number of lines in the file?

Reply:
Well. We plan to add metadata for graphs (number of vertices and edges). So, can we use this data here?

Reply:
Yes, of course. I can add additional flags and corresponding filters when the metadata appears.